### PR TITLE
Add Rogando story

### PR DESCRIPTION
**Pull request by xmanu2015**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1930)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: espanol/rogando/rogando.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Dej� caer su cayado en el suelo, se arrodill� junto a Betito y or� al cielo para que Dios mande un ej�rcito de �ngeles.
```
Comment (casey-collab) [inline espanol/rogando/rogando.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by xmanu2015** [comment]
Hi @casey-collab, I fix it, please review this

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by xmanu2015** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**